app/views/admin/basic_projects/_details.slim

```diff
@@ -11,12 +11,7 @@ section.details class=(@basic_project.valid? ? "show-view" : " edit-view")
         a href=admin_basic_project_path(@person) = t("cancel_edit")
     - if policy(@basic_project).destroy?
       = link_to [:admin, @basic_project], method: :delete,
-          data: { \
-            confirm: t("basic_project.confirm_deletion"),
-            "confirm-title": t("basic_project.delete"),
```

Comment:
What happens to the modal titles and modal custom button text with these changes? It looks like those customizations are lost. Cancel and confirm button text are not needed necessarily, as across files, those look to be the same messages so they don't need to be defined in multiple places. But what about "confirm-title"?

Reply:
the responsible gem `twitter-bootstrap-rails-confirm` is not compatible with rails 5 yet. This  also brought up the question of whether we needed fancy confirm boxes

Reply:
Okay. Thanks!

Reply:
It's more important to be secure than to have fancy confirm boxes. So these changes are fine.

Reply:
This did end up causing a bug. So we'll look into it.